File: Beginner/Id and Ship (FLOW010)/ship.py
t = int(input())

while t:
    c = input()
    if c=='b' or c=='B':
        print('BattleShip')
    if c=='c' or c=='C':
        print('Cruiser')
    if c=='D' or c=='d':
        print('Destroyer')
    if c=='F' or c=='f':
        print('Frigate')
    t = t-1

File: Beginner/Id and Ship (FLOW010)/ship.cpp
#include <iostream>
using namespace std;

int main() {
    int t;
    cin >> t;
    while(t--){
        char c;
        cin >> c;
        
        if(c=='B' || c=='b') {
            cout<<"BattleShip"<<endl;
        }
        else if(c=='C' || c=='c') {
            cout<<"Cruiser"<<endl;
        }
        else if(c=='D' || c=='d') {
            cout<<"Destroyer"<<endl;
        }
        else {
            cout<<"Frigate"<<endl;
        }
    }
	return 0;
}